src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -60,20 +70,32 @@
     private final DialogService dialogService;
     private final BibEntry entry;
     private final TaskExecutor taskExecutor;
+    private final FileDirectoryPreferences fileDirectoryPreferences;
+    private final CleanupPreferences cleanupPreferences;
+    private final LayoutFormatterPreferences layoutFormatterPreferences;
+    private final XmpPreferences xmpPreferences;
+    private final String fileNamePattern;
 
+    @Deprecated
```

Comment:
Please add a JavaDoc pointing to the method one should use from now on.

Reply:
Good observation. Fixed.